openapi/roll/tool_test.go

```diff
@@ -60,7 +60,7 @@ func TestRegenerateExamples(t *testing.T) {
 	target := "../.."
 	pass := render.NewPass(target, s.ServicesExamples(), map[string]string{
 		".codegen/examples_test.go.tmpl": "service/{{.Package}}/{{.SnakeName}}_usage_test.go",
-	})
+	}, []string{})
```

Comment:
Unrelated to the changes but do we plan to have the above test or we probably should remove it?
`"universe/bazel-bin/openapi/all-internal.json"`

`````
func TestOptimize(t *testing.T) {
	t.Skip()
	ctx := context.Background()
	home, _ := os.UserHomeDir()
	batch, err := code.NewFromFile(ctx, filepath.Join(home,
		"universe/bazel-bin/openapi/all-internal.json"))
	require.NoError(t, err)
	s, err := NewSuite("../../internal")
	require.NoError(t, err)
	err = s.OptimizeWithApiSpec(batch)
	require.NoError(t, err)
}
`````

Reply:
That test is useful for debugging issues with example optimization. A developer can temporarily remove the skip at the start to easily run the example optimization.